apps/e2e/ag-grid-storybook-e2e/src/e2e/data-entry-grid.component.cy.ts

```diff
@@ -20,20 +20,20 @@ describe('ag-grid-storybook', () => {
           .end()
 
           // Activate a date field without the calendar.
-          .get('#editDate div[row-id="aparilu01"] > div[col-id="birthday"]')
+          .get('#editDate div[row-id="bankser01"] > div[col-id="birthday"]')
```

Comment:
The `name` column is now sorted to show the juxtaposition of the sort icon with the inline help button, but that changes the data and affects most of the visual regression tests and a few of these Cypress assertions.